examples/vite-demo-vanilla-bundle/src/examples/example09.ts

```diff
@@ -131,6 +145,16 @@ export default class Example09 {
           enableCount: this.isCountEnabled, // add the count in the OData query, which will return a property named "__count" (v2) or "@odata.count" (v4)
           enableSelect: this.isSelectEnabled,
           enableExpand: this.isExpandEnabled,
+          filterQueryOverride: (args) => {
+            const { fieldName, columnDef, columnFilterOperator, searchValue } = args;
```

Comment:
while reimplementing this in GraphQL, I noticed that these 2 lines can be simplified into 1 line by simply deconstructing the object directly in the function argument

```ts
filterQueryOverride: ({ fieldName, columnDef, columnFilterOperator, searchValue }) => {
```